src/app-layer-htp.c

```diff
@@ -488,6 +521,8 @@ struct {
 /*    { "Invalid authority port", HTTP_DECODER_EVENT_INVALID_AUTHORITY_PORT}, htp no longer returns this error */
     { "Request buffer over", HTTP_DECODER_EVENT_REQUEST_FIELD_TOO_LONG},
     { "Response buffer over", HTTP_DECODER_EVENT_RESPONSE_FIELD_TOO_LONG},
+    { "C-T multipart/byteranges in responses not supported", HTTP_DECODER_EVENT_RESPONSE_MULTIPART_BYTERANGES},
```

Comment:
Suggest `byte-ranges` instead

Reply:
This table is used to match against strings set in the libhtp library, so we'll have to use what libhtp does.

Reply:
And this is a back port, so we should first fix master I guess...